docs/fsharp/language-reference/pattern-matching.md

```diff
@@ -37,11 +37,11 @@ Supported patterns are shown in the following table. At run time, the input is t
 |Identifier pattern|A case value of a discriminated union, an exception label, or an active pattern case|`Some(x)`<br /><br />`Failure(msg)`|
 |Variable pattern|*identifier*|`a`|
 |`as` pattern|*pattern* as *identifier*|`(a, b) as tuple1`|
-|OR pattern|*pattern1* &#124; *pattern2*|`([h] &#124; [h; _])`|
```

Comment:
Because of how markdown is rendered in tables, it will actually have to be this:

```<code>([h] &#124; [h; _])</code>```